locust_plugins/jmeter_listener.py

```diff
@@ -41,9 +43,12 @@ def __init__(
         self.timestamp_format = timestamp_format
         # how many records should be held before flushing to disk
         self.flush_size = flush_size
-        # results filename format
-        self.results_timestamp_format = "%Y_%m_%d_%H_%M_%S"
-        self.results_filename = f"results_{datetime.fromtimestamp(time()).strftime(self.results_timestamp_format)}.csv"
+        if results_filename == "":
+            # results filename format
+            self.results_timestamp_format = "%Y_%m_%d_%H_%M_%S"
```

Comment:
this probably should never have been an object variable (but it looks even weirder now that it is only optionally set). remove or make it a function variable